flask2postman/__init__.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+from importlib import import_module
+
+from .utils import init_virtualenv
+from .postman_v1 import Collection as CollectionV1
+from .postman import Collection
```

Comment:
It's not a big deal, but can we:
* use absolute imports?
* find something more consistent? For example `postman.v1` and `postman.v2`

Reply:
What if, for some reason, we want to support Postman 2.0 at the same time, or any other 2.x version? :slightly_smiling_face: